spec/controllers/admin_request_controller_spec.rb

```diff
@@ -55,18 +55,21 @@
 
   describe 'when fully destroying a request' do
 
-    it 'expires the file cache for that request' do
```

Comment:
AFAICT, the controller can no longer see whether the expire message has been sent